lib/segment/src/vector_storage/simple_sparse_vector_storage.rs

```diff
@@ -175,6 +155,11 @@ impl VectorStorage for SimpleSparseVectorStorage {
         self.total_vector_count
     }
 
+    fn available_size_in_bytes(&self) -> usize {
+        // multiply by 2 to account for indices & values
+        self.total_sparse_size * 2
```

Comment:
Discussion point. Do we need to multiply by two here? @agourlay
Also, is the total size correct here?